src/utils/AttributionManager.js

```diff
@@ -553,17 +555,18 @@ export class SnapshotAttributionManager extends ObservableV2 {
   /**
    * @param {Snapshot} prevSnapshot
    * @param {Snapshot} nextSnapshot
+   * @param {Array<import('./IdMap.js').AttributionItem<any>>} [attrs]
    */
-  constructor (prevSnapshot, nextSnapshot) {
+  constructor (prevSnapshot, nextSnapshot, attrs = [createAttributionItem('change', '')]) {
```

Comment:
I know used the `change` attribution here before, but I think this should just be empty.